front/app/containers/Admin/sideBar/index.tsx

```diff
@@ -144,9 +143,10 @@ export type NavItem = {
   iconName: IconNames;
   message: string;
   featureName?: TAppConfigurationSetting;
-  isActive: (pathname: string) => boolean;
+  isActive?: (pathname: string) => boolean;
```

Comment:
Did you search the code base for usage? E.g. DesktopNavbarItem still seems to have it.

If after all removal it's not used anymore, you can delete this prop?

Reply:
Good catch, that can be removed too. I found a few other instances and removed them and the functionality seems to still work